applications/GeoMechanicsApplication/geo_mechanics_application.cpp

```diff
@@ -310,6 +328,20 @@ void KratosGeoMechanicsApplication::Register() {
     KRATOS_REGISTER_VARIABLE( BULK_MODULUS_SOLID )
     KRATOS_REGISTER_VARIABLE( BULK_MODULUS_FLUID )
 
+    KRATOS_REGISTER_VARIABLE( HEAT_CAPACITY_WATER )
+    KRATOS_REGISTER_VARIABLE( HEAT_CAPACITY_SOLID )
```

Comment:
This refers to specific heat capacity ( as opposed to volumetric heat capacity ). That could reflect in the naming.

Reply:
SPECIFIC_HEAT_CAPACITY_WATER 
SPECIFIC_HEAT_CAPACITY_SOLID